cl/search/models.py

```diff
@@ -2144,6 +2171,72 @@ class Meta:
         ordering = ["position"]
 
 
+class Courthouse(models.Model):
+    """A class to represent the physical location of a court."""
+
+    court = models.ForeignKey(
+        Court,
+        help_text="The court object associated with this Courthouse.",
+        related_name="courts",
+        on_delete=models.CASCADE,
+    )
+    court_seat = models.BooleanField(
+        help_text="Is this the seat of the Court.",
+        default=False,
+        null=True,
+    )
+    building_name = models.TextField(
+        verbose_name="Courthouse building name.",
+        help_text="Ex. John Adams Courthouse",
+        blank=True,
+    )
+    address1 = models.TextField(
+        help_text="The normalized address1 of the courthouse.",
+        db_index=True,
+        blank=True,
+    )
+    address2 = models.TextField(
+        help_text="The normalized address2 of the courthouse.",
+        db_index=True,
+        blank=True,
+    )
+    city = models.TextField(
+        help_text="The normalized city of the organization.",
+        db_index=True,
+        blank=True,
+    )
+    county = models.TextField(
+        help_text="The county, if any, where the court resides.",
+        blank=True,
+    )
+    state = USPostalCodeField(
+        help_text="The two-letter USPS postal abbreviation for the "
+        "organization w/ obsolete state options.",
+        db_index=True,
+        choices=USPS_CHOICES + OBSOLETE_STATES,
+        null=True,
+        blank=True,
+    )
+    zip_code = USZipCodeField(
+        help_text="The zip code for the organization, XXXXX or XXXXX-XXXX "
+        "work.",
+        db_index=True,
+        blank=True,
+        null=True,
+    )
+    country_code = models.CharField(
```

Comment:
Wouldn't you know it, localflavor has a useful variable for this that we can put into a `choices`. Check out ISO 3166: https://django-localflavor.readthedocs.io/en/latest/generic/#data

Reply:
I looked for it. But I guess I wouldn't as I failed to find it last week. I'll add that in.

Reply:
okay, I dove into this one a bit, but they dont provide a nice tuple associated with the country names.  I think the best option now is just to do the two countries I know we will be adding now.  UK and US ... I suppose there could be other ones but they aren't needed for state opinions I think or need a courthouse object

Reply:
Well here's a fun fact: Nobody on Github has ever used `ISO_3166_1_ALPHA2_COUNTRY_CODES`. Every usage that comes up in search is from the localflavor package itself.

Reply:
I filed a bug to see if localflavor would find this useful: https://github.com/django/django-localflavor/issues/499